src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -19,7 +19,7 @@
     "emergency name": "Aiken",
     "emergency phone": "98765431",
     "graduation year": "2030",
-    "tags" : [ "owesMoney", "friends" ]
+    "tags" : [ "owes Money", "friends" ]
```

Comment:
should we still be using camel case? would it be better if it was "owes money"

Reply:
Technically it does not matter, but it does look funny so I changed it.